scripts_ZA/compareShapesForTTbarEstim.py

```diff
@@ -0,0 +1,298 @@
+
+        single_Mll = f.Get(histoMll_name)
+        single_Mll.SetDirectory(0)
+        Mll.Add(single_Mll)
+
+        single_jet1pt = f.Get(histojet1pt_name)
+        single_jet1pt.SetDirectory(0)
+        jet1pt.Add(single_jet1pt)
+
+        single_jet2pt = f.Get(histojet2pt_name)
+        single_jet2pt.SetDirectory(0)
+        jet2pt.Add(single_jet2pt)
+
+        single_jjDR = f.Get(histojjDR_name)
+        single_jjDR.SetDirectory(0)
+        jjDR.Add(single_jjDR)
+
+        single_lep1pt = f.Get(histolep1pt_name)
+        single_lep1pt.SetDirectory(0)
+        lep1pt.Add(single_lep1pt)
+
+        single_lep2pt = f.Get(histolep2pt_name)
+        single_lep2pt.SetDirectory(0)
+        lep2pt.Add(single_lep2pt)
+
+        single_jjpt = f.Get(histojjpt_name)
+        single_jjpt.SetDirectory(0)
+        jjpt.Add(single_jjpt)
+
+    if not data:
+        Mjj.Scale(lumi)
+        Mlljj.Scale(lumi)
+        Mll.Scale(lumi)
+        jet1pt.Scale(lumi)
+        jet2pt.Scale(lumi)
+        jjDR.Scale(lumi)
+        lep1pt.Scale(lumi)
+        lep2pt.Scale(lumi)
+        jjpt.Scale(lumi)
+ 
+    list_histos.append(Mjj)
+    list_histos.append(Mlljj)
+    list_histos.append(Mll)
+    list_histos.append(jet1pt)
+    list_histos.append(jet2pt)
+    list_histos.append(jjDR)
+    list_histos.append(lep1pt)
+    list_histos.append(lep2pt)
+    list_histos.append(jjpt)
+    return list_histos
+
+
+def main():
+    list_histos_data = []
+    list_histos_ttbarMC = []
+    list_histos_singleTop = []
+
+    path_data = '/nfs/scratch/fynu/asaggio/CMSSW_8_0_30/src/cp3_llbb/ZATools/factories_ZA/plots_2017-11-10_v0_deepCSV_MllCut_METCut_for_data/slurm/output/'
+    path_MC = '/nfs/scratch/fynu/asaggio/CMSSW_8_0_30/src/cp3_llbb/ZATools/factories_ZA/plots_2017-11-10_v0_deepCSV_MllCut_METCut_for_MCbkg/slurm/output/'
+
+    list_histos_data = addHistos(path_data, True)
+    list_histos_ttbarMC = addHistos(path_MC, False) 
+    list_histos_singleTop = getSingleTop(path_MC)
+
+    legend = []
+    c1 = []
+    c2 = []
+    c3 = []
+    pad1 = []
+    pad2 = []
+    subtractSingleTop = False
+
+    for i in range(0, len(list_histos_data)):
+
+        norm_data = list_histos_data[i].Integral()
+        norm_ttbar = list_histos_ttbarMC[i].Integral()
+        norm_singletop = list_histos_singleTop[i].Integral()
+
+        print norm_data
+        print norm_ttbar
+        print norm_singletop
+
+
+        c1.append(TCanvas("c1","c1",600,600))
+        pad1.append(TPad("pad1","pad1",0,0.3,1,0.9))
+        pad1[i].SetBottomMargin(0)
+        pad1[i].Draw()
+        pad1[i].cd()
+        list_histos_data[i].SetStats(0)
+        list_histos_data[i].SetMarkerColor(ROOT.kRed)
+        list_histos_data[i].SetMarkerStyle(20)
+        list_histos_ttbarMC[i].SetMarkerColor(ROOT.kBlue)
+        list_histos_ttbarMC[i].SetMarkerStyle(22)
+        # Subtract the SingleTop background
+        if subtractSingleTop:
+            list_histos_data[i].Add(list_histos_singleTop[i], -1)
```

Comment:
you should recompute `norm_data` here also to end up with the right normalisation no ?